src/libcrun/container.c

```diff
@@ -994,6 +994,11 @@ libcrun_container_kill_all (libcrun_context_t *context, const char *id, int sign
   ret = libcrun_cgroup_killall_signal (status.cgroup_path, signal, err);
   if (UNLIKELY (ret < 0))
     return ret;
+
+  /* Temporary fix to kill the main process in any case.  Drop it once read_all_pids
+     can read all the pids from the subdirectories.  */
+  kill (status.pid, SIGKILL);
```

Comment:
Should this be  kill (status.pid, signal);

Reply:
yes, thanks for noticing it.  Fixed now